docs/cli-reference.md

```diff
@@ -0,0 +1,589 @@
+* `login` — Manage your login to the SpacetimeDB CLI
+* `logout` — 
+* `init` — Initializes a new spacetime project. WARNING: This command is UNSTABLE and subject to breaking changes.
+* `build` — Builds a spacetime module.
+* `server` — Manage the connection to the SpacetimeDB server. WARNING: This command is UNSTABLE and subject to breaking changes.
+* `subscribe` — Subscribe to SQL queries on the database. WARNING: This command is UNSTABLE and subject to breaking changes.
+* `start` — Start a local SpacetimeDB instance
+* `version` — Manage installed spacetime versions
+
+###### <b>Options:</b>
+
+* `--root-dir <ROOT_DIR>` — The root directory to store all spacetime files in.
+* `--config-path <CONFIG_PATH>` — The path to the cli.toml config file
+
+
+
+## spacetime publish
+
+Create and update a SpacetimeDB database
+
+**Usage:** `spacetime publish [OPTIONS] [name|identity]`
+
+Run `spacetime help publish` for more detailed information.
+
+###### <b>Arguments:</b>
+
+* `<NAME|IDENTITY>` — A valid domain or identity for this database
+
+###### <b>Options:</b>
+
+* `-c`, `--delete-data` — When publishing to an existing database identity, first DESTROY all data associated with the module
+* `--build-options <BUILD_OPTIONS>` — Options to pass to the build command, for example --build-options='--skip-println-checks'
+
+  Default value: ``
+* `-p`, `--project-path <PROJECT_PATH>` — The system path (absolute or relative) to the module project
+
+  Default value: `.`
+* `-b`, `--bin-path <WASM_FILE>` — The system path (absolute or relative) to the compiled wasm binary we should publish, instead of building the project.
+* `--anonymous` — Perform this action with an anonymous identity
+* `-s`, `--server <SERVER>` — The nickname, domain name or URL of the server to host the database.
+* `-y`, `--yes` — Run non-interactively wherever possible. This will answer "yes" to almost all prompts, but will sometimes answer "no" to preserve non-interactivity (e.g. when prompting whether to log in with spacetimedb.com).
+
+
+
+## spacetime delete
+
+Deletes a SpacetimeDB database
+
+**Usage:** `spacetime delete [OPTIONS] <database>`
+
+Run `spacetime help delete` for more detailed information.
+
+
+###### <b>Arguments:</b>
+
+* `<DATABASE>` — The name or identity of the database to delete
+
+###### <b>Options:</b>
+
+* `-s`, `--server <SERVER>` — The nickname, host name or URL of the server hosting the database
+* `-y`, `--yes` — Run non-interactively wherever possible. This will answer "yes" to almost all prompts, but will sometimes answer "no" to preserve non-interactivity (e.g. when prompting whether to log in with spacetimedb.com).
+
+
+
+## spacetime logs
+
+Prints logs from a SpacetimeDB database
+
+**Usage:** `spacetime logs [OPTIONS] <database>`
+
+Run `spacetime help logs` for more detailed information.
+
+
+###### <b>Arguments:</b>
+
+* `<DATABASE>` — The name or identity of the database to print logs from
+
+###### <b>Options:</b>
```

Comment:
The h6 + <b> tag is an odd choice, I assume this is a styling workaround?

Reply:
"Yes", but what's autogenerated is `###### **Options:**`.

It does look a bit better when it's bolded, so I switched `**` for a `<b>` tag. I can change it for e.g. `<h6><b>`, but then I'd want to re-test that it looks as expected.

Reply:
`**` is bold though.

I just mean why not just bold it? Why the tiny header?

Reply:
I don't know, but that's what's generated. I just swapped the markdown bold (`**`) that didn't work for an HTML bold (`<b>`) that did.

We can do further post-processing to remove that if you feel strongly about it. My hope was to remove the hacky `sed`s if/when the generated stuff just starts working.

Reply:
Alright, I'm willing to fix this/change this in a future PR. This is an improvement in my mind.